### ISSUE TITLE
test accuracy?

### ISSUE THREAD

**Issue by iliadsouti** [rishizek/tensorflow-deeplab-v3#4]
Thanks for your great project. I wonder what is the test set accuracy?

**Comment by rishizek** [comment]
Hi @iliadsouti , Thanks for interesting in the repo.
 I also wonder. But the test set mIoU is not easily computable because I don't have the test set grandtruth labels and probably need to submit the evaluated result to PASCAL VOC Challenge performace evaluation server. It should be similar value to the validation set and I don't feel necessity to spend time for that.

**Comment by iliadsouti** [comment]
Hi,
In tensorflow implmentation of deeplab-v2 there is around 10% performance gap in test/validation. Some images may be both present in validation/training set. Refer to this [link](http://home.bharathh.info/pubs/codes/SBD/download.html). You will need to submit the result to test server. That's a standard cross implmentation validation.

**Comment by rishizek** [comment]
I see.  OK. I'll try when I have time. Thank you for your suggestion.
